crates/swc_ecma_transforms_compat/tests/es2015_destructuring.rs

```diff
@@ -180,12 +180,105 @@ test!(
 ref = [], ref1 = ref[0], code = ref1 === void 0 ? 1 : ref1, ref;"
 );
 
+test!(
+    syntax(),
+    |_| tr(),
```

Comment:
Can you change the definition of `tr` to 

```rust
fn tr() -> impl Fold {
    chain!(resolver(), destructuring(Config { loose: true }))
}
```

?

This is to match actual behavior of `@swc/core`.

Reply:
done.

Reply:
You're right. It changes a test result to be more in line with the actual behavior.